ulauncher/ui/windows/ulauncher_window.py

```diff
@@ -34,6 +34,18 @@ def __init__(self, **kwargs: Any) -> None:  # noqa: PLR0915
         width_request = int(self.settings.base_width)
         height_request = -1
 
+        # Check for potential fractional scaling
+        if not IS_X11_COMPATIBLE:
+            if monitor := get_monitor(self.settings.render_on_screen != "default-monitor"):
+                monitor_geometry = monitor.get_geometry()
+                monitor_scale = monitor.get_scale_factor()
+                logger.info(
+                    "Wayland mode detected. Monitor: %sx%s, Scale factor: %s",
+                    monitor_geometry.width,
+                    monitor_geometry.height,
+                    monitor_scale,
+                )
```

Comment:
All this code does is log a message. `monitor_geometry` and `monitor_scale` is not used for anything.